packages/text-input/src/TextInput/TextInput.tsx

```diff
@@ -152,109 +123,37 @@ const TextInput = React.forwardRef<HTMLInputElement, TextInputProps>(
       );
     }
 
-    const shouldRenderOptionalText =
-      state === State.None && !disabled && optional;
-
     return (
-      <div
-        className={cx(
-          wrapperStyle,
-          getWrapperFontSize(sizeVariant, baseFontSize),
-          className,
-        )}
+      <FormField
+        label={label}
+        description={description}
+        errorMessage={errorMessage}
+        state={state}
+        size={sizeVariant}
+        disabled={disabled}
+        baseFontSize={baseFontSize}
+        darkMode={darkMode}
+        className={className}
+        id={id}
+        optional={optional}
```

Comment:
Maybe this was discussed while I was out, but is baking in `optional` into FormField the best choice? Isn't this `optional` text only in TextInput?